### PR TITLE
Saas 13.2 view create jke

### PR DESCRIPTION
**Pull request by JKE-be**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by JKE-be** [comment]
@robotoo rebase-ff

File: addons/website/models/ir_ui_view.py [hunk 1]
```diff
@@ -40,8 +40,9 @@ def _get_pwd(self):
     def _set_pwd(self):
         crypt_context = self.env.user._crypt_context()
         for r in self:
-            r.sudo().visibility_password = crypt_context.encrypt(r.visibility_password_display)
-            r.visibility = r.visibility  # double check access
+            if r.type == 'qweb' and r.visibility_password_display:  # empty if no password and not updated
+                r.sudo().visibility_password = crypt_context.encrypt(r.visibility_password_display)
```
Comment (xmo-odoo) [inline addons/website/models/ir_ui_view.py:44]: If visibility_password_display is empty here doesn't mean this is the result of explicitly trying to empty it? Should it really just be ignored?

File: odoo/addons/base/models/ir_ui_view.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ class View(models.Model):
                        help="""This field should be used when accessing view arch. It will use translation.
                                Note that it will read `arch_db` or `arch_fs` if in dev-xml mode.""")
     arch_base = fields.Text(compute='_compute_arch_base', inverse='_inverse_arch_base', string='Base View Architecture',
-                            help="This field is the same as `arch` field without translations")
+                            help="This field is the same as `arch` field without translations", required=1)
```
Comment (xmo-odoo) [inline odoo/addons/base/models/ir_ui_view.py:224]: `arch_base` is also a computed field, the required should be on `arch_db`.

**Comment by JKE-be** [comment]
@robodoo rebase-ff

**Comment by JKE-be** [comment]
@robodoo r+

Ty xavier

**Comment by JKE-be** [comment]
@robodoo r-

Duplicate label to check :/

**Comment by JKE-be** [comment]
@robodoo r+
@fw-bot r+

**Comment by JKE-be** [comment]
@robodoo r+

Or maybe I'm fired but didn't yet know it ?

**Pull request closed without merging**